platform_versioned_docs/version-23.4/cli/commands.mdx

```diff
@@ -0,0 +1,1027 @@
+
+  Run `tw members leave -o <organization_name>` to be removed from the given organization's members.
+
+</details>  
+<details>
+  <summary>**Workspaces**</summary>
+
+  Run `tw workspaces -h` to view supported workspace operations.
+
+  Workspaces provide the context in which a user launches workflow executions, defines the available resources, and manages who can access those resources. Workspaces contain pipelines, runs, actions, datasets, compute environments, credentials, and secrets. Access permissions are controlled with participants, collaborators, and teams.
+
+  See [User workspaces][user-workspaces] for more information.
+
+  :::note
+  This section assumes that you already have access to an organization in Seqera Platform.
+  :::
+
+  ### Add a workspace
+
+  Run `tw workspaces add -h` to view the required and optional fields for adding your workspace.
+
+  In the example below, we create a shared workspace to be used for sharing pipelines with other private workspaces. See [Shared workspaces][shared-workspaces] for more information.
+
+  ```console
+  tw workspaces add --name=shared-workspace --full-name=shared-workspace-for-all  --org=my-tower-org --visibility=SHARED
+
+    A 'SHARED' workspace 'shared-workspace' added for 'my-tower-org' organization
+  ```
+
+  :::note
+  By default, a workspace is set to private when created.
+  :::
+
+  ### List workspaces
+
+  List all the workspaces in which you are a participant:
+
+  ```console
+  tw workspaces list                      
+
+    Workspaces for default user:
+
+      Workspace ID    | Workspace Name   | Organization Name | Organization ID 
+      -----------------+------------------+-------------------+-----------------
+      26002603030407  | shared-workspace | my-tower-org      | 04303000612070  
+  ```
+
+</details>
+<details>
+  <summary>**Participants**</summary>
+
+  Run `tw participants -h` to view supported participant operations.
+
+  Manage workspace participants. 
+
+  ### List participants
+
+  ```console
+  tw participants list
+
+    Participants for 'my-tower-org/shared-workspace' workspace:
+
+      ID             | Participant Type | Name                        | Workspace Role 
+      ----------------+------------------+-----------------------------+----------------
+      45678460861822 | MEMBER           | user (user@mydomain.com) | owner          
+  ```
+
+  ### Add participants
+
+  Run `tw participants add -h` to view the required and optional fields for adding a participant.
+
+  To add a new _collaborator_ to the workspace, use the `add` subcommand. The default role assigned to a _collaborator_ is `Launch`.
+
+  See [Participant roles][participant-roles] for more information.
+
+  ```console
+  tw participants add --name=collaborator@mydomain.com --type=MEMBER                           
+
+    User 'collaborator' was added as participant to 'shared-workspace' workspace with role 'launch'
+  ```
+
+  ### Update participant roles
+
```

Comment:
Does this require a level of permissions to execute this command (like `MAINTAIN` role and above)?